src/modules/auth/ui/views/sign-in-view.tsx

```
@@ -0,0 +1,189 @@
+
+type Props = {};
+
+// =>> SIGN-IN FORM VALIDATION SCHEMA
+const formSchema = z.object({
+  email: z.string().email(),
+  password: z.string().min(1, {
+    message: "Password is required",
+  }),
+});
+
+const SignInView = ({}: Props) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)